test/typings/graphql.test-d.ts

```diff
@@ -50,3 +51,22 @@ graphql.operation<
 >((req, res, ctx) => {
   return res(ctx.data({ key: 'pass' }))
 })
+
+// Supports "DocumentNode" as the GraphQL operation name.
+const getUser = parse(`
+  query GetUser {
+    user {
+      firstName
+    }
+  }
+`)
+graphql.query(getUser, (req, res, ctx) => res(ctx.data({})))
```

Comment:
@longility, we use this TypeScript module to test types. It's going to be compiled during the test run and if there are any type of violations, the compilation will fail. We can expect failures to test negative scenarios via `@ts-expect-error`. This way we make sure there are no type regressions throughout the releases.

Reply:
I did not follow too much on this. Sorry for lack of experience and/or context. Can you guide me or elaborate and is there a specific task or work you were thinking?

Reply:
Is that the commit you did?